package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@across-protocol/sdk-v2",
   "author": "UMA Team",
-  "version": "0.14.12",
+  "version": "0.14.13",
```

FWIW I just bumped the version so you probably want to go to 0.14.14: https://github.com/across-protocol/sdk-v2/releases/tag/v0.14.13
```suggestion
  "version": "0.14.14",
```

---

Rebased